src/tokens.ts

```diff
@@ -27,6 +27,16 @@ export const TOKEN_SYMBOLS_MAP = {
     },
     coingeckoId: "arbitrum",
   },
+  AZERO: {
+    name: "Aleph Zero",
+    symbol: "AZERO",
+    decimals: 18,
+    addresses: {
+      [CHAIN_IDs.ALEPH_ZERO]: "0xb7Da55D7040ef9C887e20374D76A88F93A59119E",
```

Comment:
This is the wrapped variant. Address taken from https://github.com/Cardinal-Cryptography/wAZERO-evm and confirmed w/ the team.